docker-compose.yaml

```diff
@@ -138,15 +130,16 @@ services:
       - sys_nice
     environment:
       - CACHED_GRPC_ENDPOINT=cached:8001
-      - CACHED_HEALTH_ENDPOINT=cached:8000
```

Comment:
Is removing the Health endpoint intentional?

Reply:
If so, can you explain.

Reply:
Yes, I wanted to simplify and consolidate `*_HTTP_ENDPIONT` and `*_HEALTH_ENDPOINT` variables into one. @jdye64 can correct me, but I think the original intention of the readiness check was to use the `*_HTTP_ENDPOINT` variables, parse them, and append `/v1/ready` to get the healthcheck endpionts. But we ended up adding `*_HEALTH_ENDPOINT` variables because we were not ready to make the switch to using http endpoints and moved back to having only grpc variables. Since this PR adds the http endpoint variables for all the NIMs, we can get the readiness endpoints by parsing the http endpoint variables.